### PR TITLE
Add a title to the analysis request form

### PR DESCRIPTION
**Pull request by tomodwyer**

### CODE REVIEW

File: assets/src/scripts/interactive/pages/purpose.jsx [hunk 1]
```diff
@@ -54,24 +54,26 @@ function Purpose() {
           <Textarea
             characterCount
             className="mb-6"
-            hintText="We should provide some text here to explain"
+            hintText="This title will be added to the top of your generated report. You will be able to change it later."
```
Comment (lucyb) [inline assets/src/scripts/interactive/pages/purpose.jsx:57]: How about this to provide more detail? It definitely needs some more work though.

> The title will be shown at the top of the generated report. It should provide a short summary to anyone reading the report about what the analysis shows. For example: "How the COVID-19 pandemic has affected the number of rescue packs being prescribed for COPD patients".
> You will be able to change this after the report is generated.

**Review by lucyb — APPROVED** [review]
:tada:

File: interactive/views.py [hunk 2]
```diff
@@ -19,35 +18,52 @@
...
+        return ""
+
+
 class AnalysisRequestCreate(View):
-    form_class = AnalysisRequestForm
     get_opencodelists_api = staticmethod(_get_opencodelists_api)
 
-    def build_analysis(self, *, form_data, project):
-        raw = json.loads(form_data)
+    def build_analysis(self, *, data, project):
```
[the remark this answers is not in the document]
Comment (ghickman) [reply interactive/views.py:52]: I wondered exactly the same thing, we could pass in the form's cleaned data as kwargs to `submit_analysis` and let it handle the various parts 👍

[the remark this answers is not in the document]
Comment (lucyb) [reply interactive/views.py:52]: That means the view doesn't know about a v2.analysis object, which seems like a good thing. Having submit.py being the boundary between the two systems feels right to me.

**Pull request merged**